core/utils.py

```diff
@@ -623,6 +623,20 @@ def get_time_in_millisecs(datetime_obj: datetime.datetime) -> float:
     return msecs + (datetime_obj.microsecond / 1000.0)
 
 
+def convert_milisecs_time_to_datetime_object(
```

Comment:
typo: milliseconds

Perhaps you can use msecs because I see that used often in the codebase.

Reply:
Ditto below.

Reply:
Done